contracts/modules/common/LimitManager.sol

```diff
@@ -91,13 +61,11 @@ abstract contract LimitManager is BaseModule {
      * @param _securityPeriod The security period.
      */
     function changeLimit(address _wallet, uint256 _newLimit, uint256 _securityPeriod) internal {
```

Comment:
Should we require that the new limit is > 0? Otherwise that effectively means transfers and token approvals are disabled and also no refunds from the TransferManager can happen. Might even be sensible to have a minimum required here?

Reply:
No. Setting the limit to 0 turns the Wallet into a normal multisig where all transfers need to be approved multiple keys (the guardians). I find that use case interesting and I know some of users are using their wallet that way.